src/components/Navbar.vue

```diff
@@ -26,6 +26,7 @@
               :href="link.url"
               :target="link.target"
             >
+              <img :src="link.logo" />
```

Comment:
It would be better to add a `v-if` to not include the image tag if not specified. Also an `alt` attribute is good for accessibility. 
```suggestion
              <img v-if="link.logo" :src="link.logo" alt="link-logo"/>
```